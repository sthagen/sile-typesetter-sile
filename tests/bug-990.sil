\begin[papersize=a6,class=book]{document}
\use[module=packages.retrograde,target=v0.15.0]
\neverindent
\define[command=open-spread]{}
\nofolios
\language[main=cs]
\chapter{Ahoj světe}

\language[main=ja]
\font[family=Noto Sans CJK JP]
\chapter{Konnichi Ha Sekai}

\language[main=tr]
\font[family=Gentium Plus]
\chapter{Selam Dünya}

\language[main=en]
\chapter{Hello World}
\end{document}
