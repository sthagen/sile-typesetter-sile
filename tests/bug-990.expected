Set paper size 	297.6377985	419.5275636
Begin page
Mx 	24.7039
My 	65.3156
Set font 	Gentium Plus;22;800;;normal;;;LTR
T	46 68 83 76 87 82 79 68 w=75.1309	(Kapitola)
Mx 	105.6824
T	20 w=10.3232	(1)
Mx 	24.7039
My 	88.1222
T	36 75 82 77 w=41.9912	(Ahoj)
Mx 	72.5418
T	86 89 629 87 72 w=47.2441	(světe)
Mx 	24.7039
My 	124.9953
Set font 	Noto Sans CJK JP;22;800;;normal;;;LTR
T	29804 w=22.0000	(第)
Mx 	46.7477
T	19 w=12.2100	(2)
Mx 	59.0014
T	29653 w=22.0000	(章)
Mx 	24.7039
My 	151.1793
Mx 	14.0360
T	44 a=14.2120	(K)
Mx 	38.7510
T	80 w=13.3320	(o)
Mx 	52.0941
T	79 w=13.4420	(n)
Mx 	65.5471
T	79 w=13.4420	(n)
Mx 	79.0002
T	74 w=6.0500	(i)
Mx 	85.0613
T	68 w=11.1980	(c)
Mx 	96.2703
T	73 w=13.3540	(h)
Mx 	109.6354
T	74 w=6.0500	(i)
Mx 	121.6209
T	41 w=16.0160	(H)
Mx 	137.6480
T	66 w=12.4080	(a)
Mx 	155.9916
T	52 w=13.1340	(S)
Mx 	169.1366
T	70 w=12.1880	(e)
Mx 	181.3357
Mx 	11.9020
T	76 a=12.1440	(k)
Mx 	193.2488
T	66 w=12.4080	(a)
Mx 	205.6678
T	74 w=6.0500	(i)
Mx 	24.7039
My 	181.1157
Set font 	Gentium Plus;22;800;;normal;;;LTR
T	37 124 79 129 80 w=58.4482	(Bölüm)
Mx 	89.0038
T	22 w=10.3232	(3)
Mx 	24.7039
My 	199.0884
T	54 72 79 68 80 w=54.3662	(Selam)
Mx 	84.9113
T	39 129 81 92 68 w=57.7822	(Dünya)
Mx 	24.7039
My 	234.1099
T	38 75 68 83 87 72 85 w=72.1338	(Chapter)
Mx 	102.6860
T	23 w=10.3232	(4)
Mx 	24.7039
My 	256.9165
T	43 72 79 79 82 w=47.5342	(Hello)
Mx 	78.0813
T	58 82 85 79 71 w=55.8809	(World)
End page
Finish
